netmiko/base_connection.py

```diff
@@ -188,8 +188,16 @@ def __init__(
         :type encoding: str
         """
         self.remote_conn = None
-        self.RETURN = "\n" if default_enter is None else default_enter
-        self.TELNET_RETURN = "\r\n"
+
+        self.TELNET_RETURN = '\r\n'
+        if default_enter is None:
+            if not 'telnet' in device_type:
```

Comment:
There is a minor linting issue here in that the linter wants this to be 'if 'telnet' not in device_type (that is why the tests are failing).